packages/@aws-cdk/aws-iotevents/lib/input.ts

```diff
@@ -67,5 +131,10 @@ export class Input extends Resource implements IInput {
     });
 
     this.inputName = this.getResourceNameAttribute(resource.ref);
+    this.inputArn = this.stack.formatArn({
```

Comment:
Actually, this should use the `getResourceArnAttribute()` helper, similarly like `inputName` uses `getResourceNameAttribute()`.

See [here](https://github.com/aws/aws-cdk/blob/0e08eebd2f13ab0da6cac7b91288845cad530192/packages/%40aws-cdk/aws-codedeploy/lib/ecs/application.ts#L73-L78) on how to use that helper.

Reply:
Can [`39a1573` (#18617)](https://github.com/aws/aws-cdk/pull/18617/commits/39a15730a7222a91f85fd3c4987e7216a3f301d6) address this comment? Or using `this.stack.formatArn()` is better instead of `arnForInput()`?